commercetools-models/src/main/java/io/sphere/sdk/products/search/ProductCategoriesIdTermFacetSearchModel.java

```diff
@@ -24,6 +24,9 @@
     @Override
     ProductCategoriesIdTermFacetSearchModel<T> withAlias(final String alias);
 
+    @Override
+    ProductCategoriesIdTermFacetSearchModel<T> countingProducts(final Boolean isCountingProducts);
```

Comment:
@lauraluiz shouldn't it be `withCountingProducts(param)` and `Boolean isCountingProducts()` ? The `is` is part of the bean getter.